### PR TITLE
Template for Container Image Page

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

File: help.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+A longer description of the container's purpose.
+
+# USAGE
+This is a free-form section that describes how the container image is used.
+Depending on how the container image was designed,
+it could describe how to use it with docker, oc, atomic, runc or other commands.
+
+To get the <container_name> container image on your local system, run the following:
+
+    docker pull hub.docker.io/<container_name>
```
Comment (eliskasl) [inline help.md]: The registry address will definitely be different in fedora world :) But that's just a nitpick.

File: README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ There are two templates within this repository on two different branches:
  * `README.md` — global documentation for the whole service
   * what is it, how to build, how to use
  * `run_test.sh` — run tests of all images with a single script
-
+ * `help.md` - Container manual page. Use the ```go-md2man``` command to convert it : `go-md2man -in help.md -out help.man`
```
Comment (eliskasl) [inline README.md:22]: Later, the conversion should be done by the build system.

**Review by eliskasl — APPROVED** [review]

File: help.md (hunk 1 above)
Comment (phracek) [reply help.md]: @eliskasl Thx, for the feedback, I can change it like ```docker pull <registry>/<container_name>```

File: README.md (hunk 2 above)
Comment (phracek) [reply README.md:22]: @eliskasl Well, I will slightly modify. Like: Container manual page. It is converted to manual page automatically by the build system.
Does it make sense?

Comment (eliskasl) [reply README.md:22]: @phracek, keep it as it is, it will take some time for the build service to be able to do it. Feel free to merge :)

**Comment by TomasTomecek** [comment]
👍

**Pull request merged**